openassessment/workflow/migrations/0003_remove_uuid_test.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workflow', '0002_remove_django_extensions'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='assessmentworkflow',
+            name='uuid',
+        ),
+    ]
```

Comment:
Worth noting: this migration ends up being unreversible, due to the default value `uuid()` being identical when run a bunch of times at once. There ought to be a better way to do that, maybe I'll mess with the default value in the initial "prepare for removal" work